articles/roadmap-preview-april-2025.md

```diff
@@ -0,0 +1,41 @@
+# Roadmap preview, April 2025
+
+<!--
+<div purpose="embedded-content">
+   <iframe src="TODO" allowfullscreen></iframe>
```

Comment:
Hey @Drew-P-drawers the roadmap preview video is here in drive: https://drive.google.com/drive/folders/1yXtLHOg19p-kMQMaGEPZcSU9rAbKQ7sa

Can you please help me clean the video, post it to YouTube, and add the embed in this guide? Heads up that I show some sensitive info in the video like our dogfood enroll secret and some host serial numbers. I think we want to blur those out if possible.

Reply:
Going to start working on this now @noahtalerman